helm/templates/namespace.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if not (has "*" .Values.targetNamespaces) }}
+{{- range .Values.targetNamespaces }}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ . | quote }}
```

Comment:
If `targetNamespaces` includes namespaces that already exist, the chart will fail, making the installation of connaissuer in an existing cluster a headache. On the other hand I do realize that we need the labels for matching to the webhook. 

My idea would be to add something like a `freshInstallation` field (boolean) to the `values.yaml`, indicating whether connaisseur is being installed in an empty cluster and can create the namespaces or not. Default would be `false`, so we do not create the namespaces ourselves, but assume their existance. Then we ideally need some kind of bash script, that can automatically update all the needed namespaces with the required labels.

It's a bit hack-y and not the clean "just do `helm install`" way, but it's the only thing I can come up with right now. Maybe you have a better idea?

Reply:
I think the wisest thing to do is to change the requirement. Let's just make `targetNamespaces` a list of new Namespaces, admitting that it will fail for existing ones with the same name. Chances are that if you are about to initialize such a security measure to a Namespace, it'd better be empty.

Reply:
Fair point.